.github/workflows/ci-macvim.yaml

```diff
@@ -46,15 +46,16 @@ jobs:
       matrix:
         include:
           - os: macos-11
-            xcode: '13.0'
             publish: true
           - os: macos-11
+            xcode: '12.5.1'
```

Comment:
Is there a reason for this Xcode version?  I was thinking maybe we should make it 11.7 for the oldest Xcode version GitHub provides.

Reply:
I simply chose the previous version, but as you said 11.7 is better since it is compatible with 10.15. I'll change it.